content/js/index.md

```diff
@@ -60,11 +60,32 @@ var skygear = require('skygear');
 #### For webpack project
 
 If you wish to use [webpack](https://webpack.github.io/) to bundle up your
-front end JavaScript code, please read [this guide](/js/guide/reactjs) about
-Skygear JS SDK integration with React.js using Babel and Webpack. You are also
-welcomed to use bundler of your choice, but bear in mind that there might be
-conditional require issues and make sure
-[Babel Polyfill](https://babeljs.io/docs/usage/polyfill/) is included.
+front end JavaScript code together with Skygear JS SDK, simply modify your
+`webpack.config.js` file to include the following lines:
```

Comment:
we should refer it as the webpack config, it is up to the user how he names the file